src/sundials/sundials_nvector.c

```diff
@@ -799,6 +802,11 @@ int N_VLinearCombinationVectorArray(int nvec, int nsum, realtype* c,
  * OPTIONAL local reduction kernels (no parallel communication)
  * -----------------------------------------------------------------*/
 
+sunindextype N_VGetLocalLength(N_Vector v)
```

Comment:
I dislike the placement of this function (it is not an "optional local reduction kernel")

Reply:
Moved it :smile: